1-js/05-data-types/02-number/article.md

```diff
@@ -203,24 +203,24 @@ alert( 0.1 + 0.2 ); // 0.30000000000000004
 
 Ouch! There are more consequences than an incorrect comparison here. Imagine you're making an e-shopping site and the visitor puts `$0.10` and `$0.20` goods into his chart. The order total will be `$0.30000000000000004`. That would surprise anyone.
 
-Why does it work like that?
+But why does this happen?
 
-A number is stored in memory in it's binary form, as a sequence of ones and zeroes. But fractions like `0.1`, `0.2` that look simple in the decimal numeric system are actually unending fractions in their binary form.
+A number is stored in memory in it's binary form, a sequence of ones and zeroes. But fractions like `0.1`, `0.2` that look simple in the decimal numeric system are actually unending fractions in their binary form.
```

Comment:
Maybe "it's" should be replaced by "its"? Or just "the binary form"?

Reply:
Yes, "it's" should be changed to "its". Removing "as" is appropriate.